src/components/base/DropDown/index.js

```diff
@@ -156,12 +161,12 @@ class DropDown extends PureComponent<Props> {
           selectedItem,
           ...downshiftProps
         }) => (
-          <Box {...getRootProps({ refKey: 'innerRef' })} horizontal relative>
+          <Wrapper shrink={shrink} {...getRootProps({ refKey: 'innerRef' })} horizontal relative>
```

Comment:
just make sure this changes does not impact any of the other usage of DropDown (plz check for all places of dropdown)

Reply:
I've checked, it's only used on the range and sort selector as far as I can tell, they work.